demos/common/python/openvino/model_zoo/model_api/models/background_matting.py

```diff
@@ -150,3 +150,36 @@ def postprocess(self, outputs, meta):
         fgr = cv2.cvtColor(cv2.resize(fgr, (w, h)), cv2.COLOR_RGB2BGR)
         pha = np.expand_dims(cv2.resize(pha, (w, h)), axis=-1)
         return fgr, pha
+
+
+class PortraitBackgroundMatting(ImageModel):
+    __model__ = 'Portrait-matting'
+
+    def __init__(self, model_adapter, configuration, preload=False):
+        super().__init__(model_adapter, configuration, preload)
+        self._check_io_number(1, 1)
+        self.output_blob_name = self._get_outputs()
+
+    @classmethod
+    def parameters(cls):
+        return super().parameters()
```

Comment:
no need in overriding method if there is no own realization
```suggestion
```

Reply:
`VideoBackgroundMatting` and `ImageMattingWithBackground` wrappers has the same method, I would leave it as it is (just to be aligned with them)